source/_integrations/nest.markdown

```
@@ -163,24 +163,78 @@
+
+2. Click **Create Topic**.
+
+3. Enter a *Topic ID* such as `home-assistant-nest`. You may leave the default settings.
+
+  ![Screenshot of OAuth confirmation](/images/integrations/nest/cloud_pubsub_create_topic.png)
+
+4. Click **Create** to create the topic.
+
+5. You now have a *Topic Name* needed by the Device Access Console and Home Assistant. The full *Topic Name* that contains your Cloud Project ID and the *Topic ID* such as `projects/<cloud console id>/topics/home-assistant-nest`.
+
+6. Next you need to give the Device Access Console permission to public to your Topic. From the Pub/Sub Topic page click **Add Principal**.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 3; Actual: 6; Style: 1/2/3

(MD029, ol-prefix)